packages/itwin/quantity-formatting/public/locales/en/QuantityFormat.json

```
@@ -42,7 +41,7 @@
       "ratioTypeLabel": "Ratio Type",
       "azimuthBase": "Azimuth Base",
       "azimuthCounterClockwise": "Enable Azimuth Counter-Clockwise",
-      "samplePreview": "Preview",
+      "samplePreview": "Try It Out",
```

we should be using sentence case

```suggestion
      "samplePreview": "Try it out",
```